python/xorbits/_mars/dataframe/groupby/rolling.py

```diff
@@ -0,0 +1,328 @@
+# Copyright 2022-2023 XProbe Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import OrderedDict
+
+import numpy as np
+import pandas as pd
+
+from ... import opcodes
+from ...core import OutputType
+from ...serialization.serializables import (
+    AnyField,
+    BoolField,
+    DictField,
+    Int32Field,
+    Int64Field,
+    KeyField,
+    Serializable,
+    StringField,
+    TupleField,
+)
+from ..operands import DataFrameOperand, DataFrameOperandMixin
+from ..utils import parse_index, validate_axis
+
+_PAIRWISE_AGG = ["corr", "cov"]
+
+
+class GroupByRolling(Serializable):
+    _input = KeyField("input")
+    _window = AnyField("window")
```

Comment:
```suggestion
    window = AnyField("window")
```

Recommend to use public directly, and the @property is not needed.